src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -192,7 +273,7 @@ class JesBackend extends Backend with LazyLogging {
   private def runWithJes(backendCall: BackendCall, command: String, jesInputs: Seq[JesParameter], jesOutputs: Seq[JesParameter]): ExecutionResult = {
     val tag = makeTag(backendCall)
     // FIXME: Ignore all the errors!
```

Comment:
This looks like the kind of thing I'd write...

Almost certainly about an Option.get or assuming a Try succeeded, or some similar scala heresy.

Since I can't see anything like that in the vicinity, the problem was probably removed but the FIXME line just wasn't tidied up with it. I suggest it's just removed.